routes/users.js

```diff
@@ -0,0 +1,6 @@
+const router = require('express').Router();
+
+router.get('/all-users', require('../controllers/getUsersController'));
+// router.use('', require(''));
```

Comment:
Since the route is already specifued to be users, maybe i a little redundant to use all-users here. Specifiying /all should suffice (/api/users/all).

Reply:
Renamed 
c310affbbb432086b970d89184f7537a5a347900